pixi_diff_to_markdown/cli.py

```diff
@@ -46,11 +46,18 @@ def main(
         ),
     ] = None,
     hide_tables: Annotated[
-        Optional[bool],
+        Optional[HideTables],
```

Comment:
i personally find `bool | Literal['auto']` nicer here but i can explore switching to cyclopts in another PR :)

Reply:
Agreed, I also found it a bit nicer. Or even the `bool | int` solution if we want to get rid of the new arguments, as it's not _that_ useful n combination with `True`/`yes`.